satellite/console/consoleweb/consoleql/schema.go

```diff
@@ -1,25 +1,17 @@
-// Copyright (C) 2019 Storj Labs, Inc.
+// Copyright (C) 2018 Storj Labs, Inc.
 // See LICENSE for copying information.
 
 package consoleql
 
 import (
-	"sync"
-
 	"github.com/graphql-go/graphql"
 
 	"storj.io/storj/satellite/console"
 	"storj.io/storj/satellite/mailservice"
 )
 
-// creatingSchemaMutex locks graphql.NewSchema method because it's not thread-safe
-var creatingSchemaMutex sync.Mutex
-
-// CreateSchema creates both type
+// CreateSchema creates a schema for satellites console graphql api
 func CreateSchema(service *console.Service, mailService *mailservice.Service) (graphql.Schema, error) {
```

Comment:
The locking still needs to happen inside CreateSchema-s since it's being called from tests as well. Meaning there's still a  potential race.